app/views/job_runs/_job_runs.html.erb

```diff
@@ -1,6 +1,9 @@
 <% @job_runs.each do |job_run| %>
     <tr>
     <td><%= link_to "#{job_prefix}#{job_run.id}", job_run, data: { turbo_frame: "_top" } %></td>
+    <% if show_project %>
+        <td><%= link_to job_run.batch_context.project_name, batch_context_path(job_run.batch_context), data: { turbo_frame: "_top" } %></td>
+    <% end %>
```

Comment:
link from job run to the project (and allow us to skip, which we will continue to do from the home page "recent history" panel)